File: leetcode beginners/minimum element after replacing with digit sum.py
class Solution:
    def minElement(self, nums: List[int]) -> int:
        mini = 36
        for _ in nums:
            count = 0
            tmp = _
            while (tmp > 0):
                count += tmp % 10
                tmp //= 10
            mini = min(count, mini)
        return mini

File: leetcode in c/Minimum element after replacing with digit sum.c
int minElement(int* nums, int numsSize) {
    int min = 36;

    for (int i = 0; i < numsSize; i++)
    {
        int count = 0;
        int tmp = nums[i];

        while (tmp > 0)
        {
            count += tmp % 10;
            tmp = tmp / 10;
        }
        min = fmin(count, min);
    }
    return min;
}